src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -140,8 +148,8 @@ def generate_MCP_1_default_config(self, filename):
     
     # Protected entities records for multiple iterations
     protected_entities_iteration_0 = {
-        'palRecords': pal_records_1,
-        'ppaRecords': ppa_record_1, 
+        'palRecords': [pal_records_1],
```

Comment:
This should not be a list of a list. Please match with the documentation in our meeting minutes. It is a waste of our time to go around and around on these issues which are very clearly documented.

Reply:
Updated,   removed from list of list, now it’s a list